### PR TITLE
update some names for registry

### PR DESCRIPTION
**Pull request by susanev**
fixing up some names for regsitry for seo and usability reasons

### CODE REVIEW

File: pkg/codegen/docs/gen.go [hunk 1]
```diff
@@ -59,8 +59,10 @@ func titleLookup(shortName string) (string, bool) {
 		"aiven":                                "Aiven",
 		"akamai":                               "Akamai",
 		"alicloud":                             "Alibaba Cloud",
+		"artifactory":				"Artifactory",
```
Comment (susanev) [inline pkg/codegen/docs/gen.go:62]:
```suggestion
		"artifactory":                             "Artifactory",
```

File: pkg/codegen/docs/gen.go [hunk 2]
```diff
@@ -59,8 +59,10 @@ func titleLookup(shortName string) (string, bool) {
...
 		"auth0":                                "Auth0",
 		"aws":                                  "AWS Classic",
+		"awsx":					"AWSx (Pulumi Crosswalk for AWS)",
```
Comment (susanev) [inline pkg/codegen/docs/gen.go:65]:
```suggestion
		"awsx":                             "AWSx (Pulumi Crosswalk for AWS)",
```

File: pkg/codegen/docs/gen.go [hunk 3]
```diff
@@ -83,6 +85,7 @@ func titleLookup(shortName string) (string, bool) {
 		"cloudamqp":                            "CloudAMQP",
 		"cloudflare":                           "Cloudflare",
 		"cloudinit":                            "cloud-init",
+		"command":				"Command (preview)",
```
Comment (susanev) [inline pkg/codegen/docs/gen.go:88]:
```suggestion
		"command":                             "Command (preview)",
```

File: pkg/codegen/docs/gen.go [hunk 4]
```diff
@@ -91,11 +94,12 @@ func titleLookup(shortName string) (string, bool) {
 		"dnsimple":                             "DNSimple",
 		"docker":                               "Docker",
 		"docker-buildkit":                      "Docker BuildKit",
+		"ec":					"ElasticCloud (EC)",
```
Comment (susanev) [inline pkg/codegen/docs/gen.go:97]:
```suggestion
		"ec":                             "ElasticCloud (EC)",
```

File: pkg/codegen/docs/gen.go [hunk 5]
```diff
@@ -59,8 +59,10 @@ func titleLookup(shortName string) (string, bool) {
 		"aiven":                                "Aiven",
 		"akamai":                               "Akamai",
 		"alicloud":                             "Alibaba Cloud",
+		"artifactory":                             "Artifactory",
```
Comment (susanev) [inline pkg/codegen/docs/gen.go:62]:
```suggestion
		"artifactory":                          "Artifactory",
```

File: pkg/codegen/docs/gen.go [hunk 6]
```diff
@@ -59,8 +59,10 @@ func titleLookup(shortName string) (string, bool) {
...
 		"auth0":                                "Auth0",
 		"aws":                                  "AWS Classic",
+		"awsx":                             "AWSx (Pulumi Crosswalk for AWS)",
```
Comment (susanev) [inline pkg/codegen/docs/gen.go:65]:
```suggestion
		"awsx":                                 "AWSx (Pulumi Crosswalk for AWS)",
```

File: pkg/codegen/docs/gen.go [hunk 7]
```diff
@@ -83,6 +85,7 @@ func titleLookup(shortName string) (string, bool) {
 		"cloudamqp":                            "CloudAMQP",
 		"cloudflare":                           "Cloudflare",
 		"cloudinit":                            "cloud-init",
+		"command":                             "Command (preview)",
```
Comment (susanev) [inline pkg/codegen/docs/gen.go:88]:
```suggestion
		"command":                              "Command (preview)",
```

File: pkg/codegen/docs/gen.go [hunk 8]
```diff
@@ -91,11 +94,12 @@ func titleLookup(shortName string) (string, bool) {
 		"dnsimple":                             "DNSimple",
 		"docker":                               "Docker",
 		"docker-buildkit":                      "Docker BuildKit",
+		"ec":                             "ElasticCloud (EC)",
```
Comment (susanev) [inline pkg/codegen/docs/gen.go:97]:
```suggestion
		"ec":                                   "ElasticCloud (EC)",
```

**Review by justinvp — CHANGES_REQUESTED** [review]

File: pkg/codegen/docs/gen.go [hunk 9]
```diff
@@ -1691,6 +1692,14 @@ func (mod *modContext) getTypes(member interface{}, types nestedTypeUsageInfo) {
...
 }
 
+type fs map[string][]byte
+
+func (fs fs) add(path string, contents []byte) {
+	_, has := fs[path]
+	contract.Assertf(!has, "duplicate file: %s", path)
+	fs[path] = contents
+}
+
```
[the remark this answers is not in the document]
Comment (susanev) [reply pkg/codegen/docs/gen.go:1702]: @justinvp I could use your help on Monday all of these came from what I thought was the right gen command to run, which was obviously wrong.

[the remark this answers is not in the document]
Comment (susanev) [reply pkg/codegen/docs/gen.go:1702]: well nevermind, i started over and its clean, going to close this in favor of https://github.com/pulumi/pulumi/pull/11525

**Pull request closed without merging**